src/sider/arbeidsuforhet/avslag/AvslagForm.tsx

```diff
@@ -0,0 +1,128 @@
+  forhandsvisningLabel: "Forhåndsvis innstillingen",
+  missingBegrunnelse: "Vennligst angi begrunnelse",
+  sendVarselButtonText: "Gi avslag",
+  avbrytButton: "Avbryt",
+};
+
+const begrunnelseMaxLength = 5000;
+
+export interface ArbeidsuforhetAvslagSkjemaValues {
+  begrunnelse: string;
+  fom: Date;
+}
+
+export const AvslagForm = () => {
+  const sendVurdering = useSendVurderingArbeidsuforhet();
+  const { getAvslagDocument } = useArbeidsuforhetVurderingDocument();
+  const formMethods = useForm<ArbeidsuforhetAvslagSkjemaValues>();
+  const {
+    register,
+    watch,
+    formState: { errors },
+    handleSubmit,
+  } = formMethods;
+
+  const submit = (values: ArbeidsuforhetAvslagSkjemaValues) => {
+    const vurderingRequestDTO: VurderingRequestDTO = {
+      type: VurderingType.AVSLAG,
+      begrunnelse: values.begrunnelse,
+      document: getAvslagDocument({
+        begrunnelse: values.begrunnelse,
+        fom: values.fom,
+      }),
+      gjelderFom: dayjs(values.fom).format("YYYY-MM-DD"),
+    };
+    sendVurdering.mutate(vurderingRequestDTO);
+  };
+
+  return (
+    <Box background="surface-default" padding="4" className="mb-2">
+      <FormProvider {...formMethods}>
+        <form onSubmit={handleSubmit(submit)} className="[&>*]:mb-4">
+          <Heading level="2" size="medium">
+            {texts.title}
+          </Heading>
+          <AvslagDatePicker />
+          <BodyShort>
+            <p>{texts.info1}</p>
+            <p>{texts.info2}</p>
+          </BodyShort>
+          <Textarea
+            {...register("begrunnelse", {
+              maxLength: begrunnelseMaxLength,
+              required: texts.missingBegrunnelse,
+            })}
+            value={watch("begrunnelse")}
+            label={texts.begrunnelseLabel}
+            error={errors.begrunnelse?.message}
+            size="small"
+            minRows={6}
+            maxLength={begrunnelseMaxLength}
+          />
+          {sendVurdering.isError && (
+            <SkjemaInnsendingFeil error={sendVurdering.error} />
+          )}
+          <BodyShort>
+            {texts.afterSendInfo.header}
+            <ul>
+              {texts.afterSendInfo.tasks.map((task, index) => (
+                <li key={index}>{task}</li>
+              ))}
+            </ul>
+          </BodyShort>
+          <BodyShort>{texts.buttonDescription}</BodyShort>
+          <ButtonRow>
+            <Button loading={sendVurdering.isPending} type="submit">
+              {texts.sendVarselButtonText}
+            </Button>
+            <Button as={Link} to={arbeidsuforhetPath} variant="secondary">
+              {texts.avbrytButton}
+            </Button>
+            <Forhandsvisning
+              contentLabel={texts.forhandsvisningLabel}
+              getDocumentComponents={() =>
+                getAvslagDocument({
+                  begrunnelse: watch("begrunnelse"),
+                  fom: watch("fom"),
+                })
+              }
+            />
```

Comment:
Har vi amplitude på denne? Tenker at veldig mange har lyst til å bruke denne på avslag 🤔 Hvis man får høy prosent bruk av denne, så er jo det et argument for å legge inn "forhåndsvisning-så-send" flyt.